learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -399,18 +427,20 @@ They have several purposes, but in general they are useful for low level access
 
 .. image:: img/visual_script39.png
 
+
 Remember that dragging a connection to empty space will help you call functions or set/get properties on these:
 
 .. image:: img/visual_script40.png
 
+
 Local Variables
 ^^^^^^^^^^^^^^^
 
-These are nodes you can use as temporary storage for your graphs. Just make sure they all have the same name and type when using them
-and they will reference the same piece of memory.
+These are nodes you can use as temporary storage for your graphs. Just make sure they all have the same name and type when using them and they will reference the same piece of memory.
```

Comment:
"temporary storage for your graphs" might be understood to mean you store your graphs in them. I don't like the phrase "piece of memory"

Reply:
As I'm not familiar how things work like this, I'm not sure what to put here.